### ISSUE TITLE
Indexing fails on planet.tsv

### ISSUE THREAD

**Issue by darrenburns** [atlasr-org/atlasr#4]
Running the `install-api-geocode-data` recipe results in an error:

```
Error: Error(Deserialize { pos: Some(Position { byte: 6844445313, line: 23872188, record: 23872187 }), err: DeserializeError { field: Some(9), kind: ParseFloat(ParseFloatError { kind: Empty }) } })

error: Recipe `install-api-geocode-data` failed on line 55 with exit code 1
```
Line 23872188 of `planet.tsv` is the culprit:

```
$ sed '23872188q;d' planet.tsv
Perrongen		node	2581279162	multiple	city_block	18.0112208	59.2795054				Södertörn		Svealand	Sweden	se	Perrongen, Älvsjö postort, Södertörn, Stockholms län, Svealand, Sweden	18.0112208	59.2795054	18.0112208	59.2795054			
```
Putting this record into a small TSV file and trying to ingest it with the indexer is a simple reproduction of the problem:
```
name	alternative_names	osm_type	osm_id	class	type	lon	lat	place_rank	importance	street	city	county	state	country	country_code	display_name	west	south	east	north	wikidata	wikipedia	housenumbers
Perrongen		node	2581279162	multiple	city_block	18.0112208	59.2795054				Södertörn		Svealand	Sweden	se	Perrongen, Älvsjö postort, Södertörn, Stockholms län, Svealand, Sweden	18.0112208	59.2795054	18.0112208	59.2795054			
```
Comparing this line with those around it, we see it's missing both the `importance` field, and the `place_rank` field. Sure enough, if those values are added, the problem no longer occurs.

Presumably the deserialiser is failing when trying to represent these missing fields as floats.

**Aside:**
Weirdly, looking at the byte offset in the error doesn't point us to the byte where the error occurred, it just points us to the start of the line with missing data:

```
$ dd if="planet.tsv" ibs=1 skip=6844445313 count=10
Perrongen	10+0 records in
0+1 records out
10 bytes transferred in 0.000062 secs (160701 bytes/sec)
```

**Comment by yanganto** [comment]
I run into the same bug.

~~> Sho Tsakha
Error: Error(Deserialize { pos: Some(Position { byte: 6844445313, line: 23872188, record: 23872187 }), err: DeserializeError { field: Some(9), kind: ParseFloat(ParseFloatError { kind: Empty }) } })
error: Recipe `install-api-geocode-data` failed on line 57 with exit code 1

**Comment by Hywan** [comment]
Fixed by #4.
